packages/api/tests/lib/rules/test-rulesHelpers.js

```diff
@@ -2271,7 +2271,7 @@ test.serial('Enabling a disabled SNS rule and passing rule.arn throws specific e
   // Should fail because a disabled rule should not have an ARN
   // when being updated
   await t.throwsAsync(updateRuleTrigger(rule, updates, t.context.testKnex),
-    null,
+    undefined,
```

Comment:
Ava upgrade changed some syntax and now prefers undefined to null.

Reply:
In general, that's a good default 👍🏻